test/integration/jit/test_jit_analytical_rosenbrock.cpp

```diff
@@ -111,7 +110,7 @@ TEST(AnalyticalExamplesJitRosenbrock, TunnelingSuperStiffButAnalytical)
 
 TEST(AnalyticalExamplesJitRosenbrock, Arrhenius)
 {
-  test_analytical_arrhenius<BuilderType<NUM_CELLS>, StateType<NUM_CELLS>>(BuilderType<NUM_CELLS>(param_two), 1e-3);
+  test_analytical_arrhenius<BuilderType<NUM_CELLS>, StateType<NUM_CELLS>>(BuilderType<NUM_CELLS>(param_two), 4e-6);
```

Comment:
How do you decide this specific-looking value `4e-6`?

Reply:
1e-6 didn't work. but 4e-6 did. Nothing fancy or scientific here